### DISCUSSION TITLE
Can I make window less than 29.714285714285715x36.57142857142857 ?

### DISCUSSION THREAD

**Discussion by ghost** [AvaloniaUI/Avalonia#5975] [Q&A]
Even if I set width, max width or height, max height less than 29.714285714285715x36.57142857142857 it doesn't apply.

**Comment by MarchingCube** [comment]
You definitely need to disable `SystemDecorations` to make window small. It seems that there is some sizing dependency between `Window` and first child since if I resize the first `Panel` contained in the window template to 0x0 then I can make the window 0x0 as well (and it will actually disappear).

**Reply by ghost** [reply]
I have already disabled SystemDecorations (SystemDecorations="None" HasSystemDecorations="False"), but window doesnt become smaller than 29.714285714285715x36.57142857142857. I set my first element (TextBlock) contained in window size to 0x0, but it doesnt help.

Code of window is
`Title="Floating Window"
 ShowInTaskbar="False"
 SystemDecorations="None" HasSystemDecorations="False"
 CanResize="False"
 Topmost="True"
 WindowStartupLocation="Manual" Width="0" Height="0" MaxWidth="0" MaxHeight="0"`
